src/ofxFaceTracker2InputInfo.h

```diff
@@ -26,23 +26,32 @@ class ofxFaceTracker2InputInfo {
     , imageRotation(imageRotation) {
         // Calculate rotation matrix
         if(imageRotation == 90){
-            rotationMatrix.makeIdentityMatrix();
-            rotationMatrix.translate(-landmarkWidth,0,0);
-            rotationMatrix.scale(((float)inputWidth / landmarkHeight), ((float)inputWidth / landmarkHeight), 1);
-            rotationMatrix.rotate(-90, 0,0,1);
+            rotationMatrix = glm::mat4(1.f);
+            rotationMatrix = glm::translate(rotationMatrix, glm::vec3((float)-landmarkWidth, 0.f, 0.f));
+            rotationMatrix = glm::scale(rotationMatrix, glm::vec3((float)inputWidth / landmarkHeight,
+                                                                  (float)inputWidth / landmarkHeight,
+                                                                  1.f));
+            rotationMatrix = glm::rotate(rotationMatrix, -90.f, glm::vec3(0.f, 0.f, 1.f));
```

Comment:
I haven't tested but isn't this supposed to be `-M_PI/2.0f` instead of `-90.f`??

Reply:
i've completely missed your comment. i think that rotate function accepts angle in degrees, not radians. i'm glad you used this for the other merge!